setup.sh

```diff
@@ -26,22 +28,16 @@ apt-get install -qy \
     vim \
     virtualenvwrapper
 
-# Add ubuntu to the passwordless sudo file.
-echo "ubuntu ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/juju-users
+# Add the JUJU_USER to the passwordless sudo file.
+echo "${JUJU_USER} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/juju-users
+
 JUJU=`which juju-1 > /dev/null && echo juju-1 || echo juju`
 # Get the version of Juju installed.
 JUJU_VERSION=`${JUJU} version`
-# The home directory for the ubuntu user.
-HOME=/home/ubuntu
-# Set some environment variables used by Juju.
-RC=${HOME}/.bashrc
-cat << EOF > $RC
-# The directory to look for charms.
-export JUJU_REPOSITORY=${HOME}
-# The path to Juju configuration files.
-export JUJU_HOME=${HOME}/.juju
-echo "Welcome to jujubox! Use the ${JUJU} command version ${JUJU_VERSION}."
-EOF
+
+# Set a welcome message in .bashrc with the exact version of Juju.
+RC=/home/$JUJU_USER/.bashrc
+echo "echo Welcome to jujubox version ${JUJU_VERSION}" >> $RC
```

Comment:
It might still be prudent to include information about how to run juju1

the binary has changed to juju-1, i was surprised when i fired up the assembled container, typed in juju and it responded with command not found :|

lameee